php/class-photonfill.php

```diff
@@ -823,6 +907,65 @@ public function swap_lazyload_classes( $content ) {
 		}
 
 		/**
+		 * Use and external url to generate a photonfill image element.
+		 * @param string. $img_url.
+		 * @param string. $img_size;
+		 * @param array. $attr. (can set alt and class)
+		 * @return string. html element.
+		 */
+		public function get_url_image( $img_url, $size, $attr = array() ) {
+			if ( ! empty( $img_url ) ) {
+				$attr = array_merge( $attr, array( 'sizes' => array(), 'srcset' => array() ) );
+				$attr['class'] = $this->get_image_classes( ( empty( $attr['class'] ) ? array() : $attr['class'] ), null, $size );
+
+				// This means post thumbnail was called w/o a size arg.
+				if ( 'post-thumbnail' == $size ) {
+					$size = 'full';
+				}
+				$image = $this->create_url_image_object( $img_url, $size );
+				$html = '';
+				$maxsize = 0;
+				$sizes = array();
+				$srcsets = array();
+				foreach ( $image['sizes'] as $breakpoint => $breakpoint_data ) {
+					$src = esc_url( $breakpoint_data['src']['url'] ) . ' ' . esc_attr( $breakpoint_data['src']['width'] . 'w' );
+					if ( ! in_array( $src, $srcsets, true ) ) {
+						$srcsets[] = $src;
+					}
+
+					$maxsize = $breakpoint_data['src']['width'] > $maxsize ? $breakpoint_data['src']['width'] : $maxsize;
+					$unit = ( ! empty( $breakpoint_data['size']['unit'] ) && in_array( $breakpoint_data['size']['unit'], $this->valid_units, true ) ) ? $breakpoint_data['size']['unit'] : 'px';
+
+					$breakpoint_size_string = '';
+					if ( ! empty( $breakpoint_data['size']['min'] ) ) {
+						$breakpoint_size_string .= '(min-width: ' . esc_attr( $breakpoint_data['size']['min'] . $unit ) . ')';
+					}
+					if ( ! empty( $breakpoint_data['size']['max'] ) ) {
```

Comment:
Do you need to check for `! empty( $breakpoint_data['size']['min']  )` here as well?  Since you use it on the next line.